homeassistant/components/sensor/pjm.py

```diff
@@ -0,0 +1,135 @@
+"""
+Support for PJM data.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.pjm/
+"""
+import asyncio
+from datetime import timedelta
+import logging
+
+import aiohttp
+import async_timeout
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['beautifulsoup4==4.6.0']
+
+_LOGGER = logging.getLogger(__name__)
+_RESOURCE = 'http://datasnapshot.pjm.com/content/InstantaneousLoad.aspx'
+
+PJM_RTO_TOTAL = "PJM RTO Total"
+
+SCAN_INTERVAL = timedelta(minutes=5)
+
+ICON_POWER = 'mdi:flash'
+
+CONF_MONITORED_FEEDS = 'monitored_feeds'
+CONF_SENSOR_TYPE = 'type'
+CONF_INSTANTANEOUS_ZONE_LOAD = 'instantaneous_zone_load'
+CONF_INSTANTANEOUS_TOTAL_LOAD = 'instantaneous_total_load'
+CONF_ZONE = 'zone'
+
+SENSOR_TYPES = {
+    CONF_INSTANTANEOUS_ZONE_LOAD: ["PJM Instantaneous Zone Load", 'MW'],
+    CONF_INSTANTANEOUS_TOTAL_LOAD: ["PJM Instantaneous Total Load", 'MW'],
+}
+
+TYPES_SCHEMA = vol.In(SENSOR_TYPES)
+
+SENSORS_SCHEMA = vol.Schema({
+    vol.Required(CONF_SENSOR_TYPE): TYPES_SCHEMA,
```

Comment:
Is there a reason to not just include this inline (rather than explicitly defining it above)?